/**:
  ros__parameters:
    # Balizas:
    beacons:
      ids: ["uwb_beacon_0", "uwb_beacon_1", "uwb_beacon_2", "uwb_beacon_3", "uwb_beacon_4"] 
      uwb_beacon_0:
        position: [14.9, 3.0, 4.5]
        noise_std: 0.1
        frame_id: "global"
      uwb_beacon_1:
        position: [-14.9, -4.0, 9.0]
        noise_std: 0.1
        frame_id: "global"
      uwb_beacon_2:
        position: [5.5, 24.9, 7.5]
        noise_std: 0.1
        frame_id: "global"
      uwb_beacon_3:
        position: [-7.0, -24.9, 8.0]
        noise_std: 0.1
        frame_id: "global"
      uwb_beacon_4:
        position: [10.0, -24.9, 5.0]
        noise_std: 0.1
        frame_id: "global"